keyboards/kopibeng/xt65/readme.md

```diff
@@ -0,0 +1,15 @@
+#-- XT65 --#
```

Comment:
```suggestion
# XT65
```

And could you add instructions on how to enter the bootloader to reflash the board?

Reply:
All done, also added image of supported layout